### ISSUE TITLE
Parsing error if decorators are applied to methods with computed properties

### ISSUE THREAD

**Issue by leonid-bauxy** [babel/babel-eslint#309]
If decorator is applied to the method with computed property key, eslint throws parse error. Is this the correct behavior?

```js
  @autobind
  [method](arg) {
      // do something
  }
```

Eslint throws error on the first `{`, saying :  `Parsing error: Unexpected token`.

Latest `eslint` and `babel-eslint`

**Comment by hzoo** [comment]
This would require a change to babylon to fix the `Unexpected token` issue if this is valid

**Comment by aaronucsd** [comment]
Borken for me as well with latest 6.0.4 version. Any updates?

**Comment by loganfsmyth** [comment]
It's not broken, it has never been supported. This is what @hzoo is trying to say above. We should close this and open an issue in Babylon, the parser Babel uses.

**Comment by leonid-bauxy** [comment]
Created an issue in bablylon repo: https://github.com/babel/babylon/issues/33. Closing this one. Thank you, guys!
